akka-actor/src/main/scala/akka/event/DeadLetterListener.scala

```diff
@@ -115,6 +117,10 @@ class DeadLetterListener extends Actor {
         val destination = if (isReal(d.recipient)) s" to ${d.recipient}" else ""
         s"Message [$messageStr]$wrappedIn$origin$destination was dropped. ${dropped.reason}. " +
         s"[$count] dead letters encountered$doneMsg. "
+      case _: UnhandledMessage =>
+        val destination = if (isReal(d.recipient)) s" to ${d.recipient}" else ""
+        s"Message [$messageStr]$wrappedIn$origin$destination was unhandled. " +
+        s"[$count] dead letters encountered$doneMsg. "
```

Comment:
if we don't log it ,then can skip this kind of string interpretation

Reply:
at this point it's always published to the eventStream